### PR TITLE
New package: scribus-dev-1.5.4

### PR DESCRIPTION
**Pull request by Kistelini**
Scribus 1.5.x is an unstable release but the last stable release is still Qt 4, this one ist Qt 5.

### CODE REVIEW

File: srcpkgs/scribus-dev/template [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Template file for 'scribus-dev'
+pkgname=scribus-dev
+version=1.5.4
+revision=1
+build_style=cmake
+configure_args="-DWANT_GRAPHICSMAGICK=1"
+patch_args="-Np1"
+hostmakedepends="pkg-config qt5-host-tools qt5-tools-devel"
+makedepends="boost-devel cairo-devel cups-devel harfbuzz-devel hunspell-devel
+	lcms2-devel libcdr-devel libfreehand-devel libgraphicsmagick-devel
```
Comment (maxice8) [inline srcpkgs/scribus-dev/template]: 1 whitespace indent

**Comment by Kistelini** [comment]
That works, thank you.

**Comment by maxice8** [comment]
I'd rather not add more unstable packages

**Comment by Hoshpak** [comment]
seconded, let's wait for upstream to publish a stable release.

**Pull request closed without merging**